packages/button/src/Button.js

```diff
@@ -147,6 +147,7 @@ const buttonStyles = (props, baseTheme) => {
     cursor: props.disabled ? 'default' : 'pointer',
     fontWeight: theme.Button_fontWeight,
     height: theme[`Button_size_${props.size}`],
+    lineHeight: 0.75, // theme.fontSize_base / parseFloat(theme.ButtonContent_fontSize_small)
```

Comment:
Without this line, the text of small and medium-sized Buttons is too low.

The meaningful lineHeight is applied to ButtonContent, so the root level lineHeight can be anything. I used Dev Tools to determine that anything below 0.87 fixes the issue. The code in the comment is my attempt to rationalize that magic number.